crates/wiggle/tests/keywords_union.witx

```diff
@@ -0,0 +1,15 @@
+(typename $union
+  (enum u8
+    $self
+    $power
+  )
+)
+
+(typename $self
+  (union $union
+    ;; A union variant that will expand to a strict keyword `Self`.
+    (field $self (@witx pointer f32))
+    ;; Oh it's true, that there's power in a union!
```

Comment:
Is it appropriate to add a joke of this kind in the source?

Reply:
Yes